src/hotspot/share/runtime/flags/jvmFlag.hpp

```diff
@@ -34,7 +34,7 @@ class outputStream;
 typedef const char* (*RangeStrFunc)(void);
 
 struct JVMFlag {
```

Comment:
I'm still getting used to git diffs, but can you change this from struct to class?  Because it looks like it does have private and public sections.

Reply:
I want to limit the changes in this PR, so I plan to do this in a future RFE, probably [JDK-8081833](https://bugs.openjdk.java.net/browse/JDK-8081833). I'll also change all the fields to private, and add accessor methods to them. And also add proper iterators for the flags (instead of walking JVMFlag::flags[] directly).

Reply:
Ok that would be fine as a further change.